src/normalize.jl

```diff
@@ -0,0 +1,77 @@
+using LinearAlgebra
+
+function rescale(tn::AbstractITensorNetwork; alg="exact", kwargs...)
+  return rescale(Algorithm(alg), tn; kwargs...)
+end
+
+function rescale(
+  alg::Algorithm"exact",
+  tn::AbstractITensorNetwork;
+  vs_to_rescale=collect(vertices(tn)),
+  kwargs...,
+)
+  logn = logscalar(alg, tn; kwargs...)
+  c = 1.0 / (exp(logn / length(vs_to_rescale)))
```

Comment:
```suggestion
  c = inv(exp(logn / length(vs_to_rescale)))
```
That is friendlier for other number types like `Float32` that are common on GPU, `1.0` is implicitly `Float64` and will promote `c` to `Float64`.